core/src/main/java/org/apache/gravitino/audit/AuditLog.java

```diff
@@ -0,0 +1,4 @@
+package org.apache.gravitino.audit;
+
+/** The interface define unified audit log schema. */
+public interface AuditLog {}
```

Comment:
Could we provide more general audit information in the interface like user, operation, resource identifier, timestamp ?